packages/react-core/src/components/ClipboardCopy/examples/ClipboardCopy.md

```diff
@@ -90,6 +97,79 @@ import { ClipboardCopy, ClipboardCopyVariant } from '@patternfly/react-core';
     ]
   }
 }} `}
-  
-</ClipboardCopy>
+</ClipboardCopy>;
+```
+
+### Inline
+
+```js
+import React from 'react';
+import { ClipboardCopy } from '@patternfly/react-core';
+
+<ClipboardCopy variant="inline">2.3.4-2-redhat</ClipboardCopy>;
+```
+
+### Inline code
```

Comment:
I would suggest we change the name of this example so the consumers are not confused.  What we are calling "inline" is what Core has as "Basic". @mcoker @mcarrano   Core should probably be updated as well for consistency.